src/ontology/hp-edit.owl

```diff
@@ -88606,7 +88606,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0008678> <http://purl.obolibrary.o
 
 # Class: <http://purl.obolibrary.org/obo/HP_0008682> (Renal tubular epithelial necrosis)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://orcid.org/0000-0002-0736-9199>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0008682> "Coagulative necrosis of tubular epithelial cells, defined as cells with increased cytoplasmic eosinophilia and nucleus that has a condensed chromatin pattern with fuzzy nuclear contour or has barely visible nuclear basophilic staining. The extent of cortical tubular necrosis is scoredsemiquantitatively as none, mild (less than 25% tubules with necrosis), moderate (25-50 percent), and severe (over 50%).")
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://orcid.org/0000-0002-0736-9199>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0008682> "Coagulative necrosis of tubular epithelial cells, defined as cells with increased cytoplasmic eosinophilia and nucleus that has a condensed chromatin pattern with fuzzy nuclear contour or has barrely visible nuclear basophilic staining. The extent of cortical tubular necrosis is scoredsemiquantitatively as none, mild (less than 25% tubules with necrosis), moderate (25-50 percent), and severe (over 50%).")
```

Comment:
@LCCarmody can you check this one again?

Reply:
original is correct

Reply:
```suggestion
AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://orcid.org/0000-0002-0736-9199>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0008682> "Coagulative necrosis of tubular epithelial cells, defined as cells with increased cytoplasmic eosinophilia and nucleus that has a condensed chromatin pattern with fuzzy nuclear contour or has barely visible nuclear basophilic staining. The extent of cortical tubular necrosis is scoredsemiquantitatively as none, mild (less than 25% tubules with necrosis), moderate (25-50 percent), and severe (over 50%).")
```